rtc.py

```diff
@@ -31,23 +42,23 @@ def get_download_url(granule):
     response.raise_for_status()
     cmr_data = response.json()
     download_url = ""
-    for product in cmr_data['feed']['entry'][0]['links']:
-        if 'data' in product['rel']:
-            download_url = product['href']
+    for product in cmr_data["feed"]["entry"][0]["links"]:
```

Comment:
If we ever don't use explicit collections, this will need to be updated as it assumes it always gets the collection with the data in it and not metadata or anything.